src/components/playback/playbackmanager.js

```diff
@@ -2143,8 +2152,26 @@ class PlaybackManager {
                 }
 
                 const apiClient = ServerConnections.getApiClient(item.ServerId);
-                apiClient.getEndpointInfo().then(function (endpointInfo) {
-                    if ((mediaType === 'Video' || mediaType === 'Audio') && appSettings.enableAutomaticBitrateDetection(endpointInfo.IsInNetwork, mediaType)) {
+
+                Promise.all([
+                    apiClient.getEndpointInfo(),
+                    apiClient.getCurrentUser()
+                ]).then(([endpointInfo, user]) => {
+                    let enableAutomaticBitrateDetection = false;
+
+                    if (mediaType === 'Video' || mediaType === 'Audio') {
+                        enableAutomaticBitrateDetection = appSettings.enableAutomaticBitrateDetection(endpointInfo.IsInNetwork, mediaType);
+
+                        if (!enableAutomaticBitrateDetection && user?.Policy
+                            && ((mediaType === 'Video' && !user.Policy.EnableVideoPlaybackTranscoding)
+                                || (mediaType === 'Audio' && !user.Policy.EnableAudioPlaybackTranscoding))) {
+                            // If transcoding is not allowed, we cannot use the specified bitrate - force bitrate detection
+                            console.warn('Transcoding is not allowed, cannot use the specified bitrate - force bitrate detection');
+                            enableAutomaticBitrateDetection = true;
```

Comment:
We could add `appSettings.enableAutomaticBitrateDetection(endpointInfo.IsInNetwork, mediaType, true);` to stop warning next play.